### PR TITLE
NIFI-6415 Add -DskipTests to README

### PR DESCRIPTION
**Pull request by andrewmlim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +128,7 @@ To run NiFi:
 If you have questions, you can reach out to our mailing list: dev@nifi.apache.org
 ([archive](http://mail-archives.apache.org/mod_mbox/nifi-dev)).
 For more interactive conversations and chat, we're also often available in IRC: #nifi on
-[irc.freenode.net](http://webchat.freenode.net/?channels=#nifi) and in #NiFi on [ASF HipChat](https://www.hipchat.com/gzh2m5YML). 
+[irc.freenode.net](http://webchat.freenode.net/?channels=#nifi) and in #NiFi on [ASF HipChat](https://www.hipchat.com/gzh2m5YML).
```
Comment (pvillard31) [inline README.md]: I think we should update the link to the Slack room instead, no? (not related to the original intention but not very harmful to change it here I think)

Comment (andrewmlim) [reply README.md]: Good idea Pierre. We list Slack, IRC and Hipchat on the following NiFi website page:

https://nifi.apache.org/mailing_lists.html

Should we remove references to IRC and HipChat on that page?  If so, I can do a separate PR for all of these related changes.

Comment (pvillard31) [reply README.md]: I don't have a strong opinion about IRC (not sure it's used a lot but there are people on the channel so...). But we should definitely remove HipChat references. Feel free to do it with another PR ;)

Comment (andrewmlim) [reply README.md]: Sounds good.  Filed https://issues.apache.org/jira/browse/NIFI-6460 for separate PR.

**Comment by mcgilman** [comment]
Thanks @andrewmlim! This has been merged to master.

**Pull request merged**